broker/broker.go

```diff
@@ -353,12 +352,19 @@ func (b *S3Broker) Bind(
 	detailc, errc := make(chan awss3.BucketDetails), make(chan error)
 	for _, bucketName := range bucketNames {
 		go func(bucketName string) {
+			b.logger.Debug("bind: goroutine: describe bucket", lager.Data{
+				instanceIDLogKey: instanceID,
+				bindingIDLogKey:  bindingID,
+				detailsLogKey:    details,
```

Comment:
From what I can see, there's nothing sensitive that we would be logging in the `BindDetails` struct, but just calling it out

Reply:
Yeah, I've taken a look and concluded the same 👍